### PR TITLE
Add cnho_stables

### PR DESCRIPTION
**Pull request by xiesh1982**

### CODE REVIEW

File: cnho_stables/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/cnho_stables/images/chain.png",
+	  "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/cnho_stables/images/chain.svg"
+          "theme": {
+            "primary_color_hex": "#708ffc"
+          }
+        }
+      ],
+      "socials": {
+        "website": "https://cnho.io/",
+        "discord": "https://discord.com/invite/bPmhYJc5rU",
```
Comment (JeremyParish69) [inline cnho_stables/assetlist.json:40]: Discord and telegram are not accepted socials at this time

File: cnho_stables/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/cnho_stables/images/chain.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/cnho_stables/images/chain.svg"
+      },
+     
+      "keywords": [
+        "CNHO"
+      ],
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/cnho_stables/images/chain.png",
```
Comment (JeremyParish69) [inline cnho_stables/assetlist.json:31]: please correct the URL paths (shouldn't have the _ underscore in 'cnho_stables')

File: cnho_stables/assetlist.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "cnhostables",
```
Comment (JeremyParish69) [inline cnho_stables/assetlist.json:3]: The chain name property here looks to have an acceptable value, but the name of the directory must match. Please change the directory name to: cnhostables (one word, no space or underscore)

File: cnho_stables/chain.json [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+      "tx_page": "https://explorer.cnho.io/transaction/${txHash}",
+      "account_page": "https://explorer.cnho.io/account/${accountAddress}"
+    }
+  ],
+  "keywords": [
+    "CNHO"
+  ],
+  "images": [
+    {
+      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/cnho_stables/images/chain.png",
```
Comment (JeremyParish69) [inline cnho_stables/chain.json:108]: The svg and png in the same image object must match. Please replace the white background of the png with a transparency layer (just as the svg has)

Comment (JeremyParish69) [reply cnho_stables/chain.json:108]: And make sure the margins match

File: cnho_stables/assetlist.json (hunk 1 above)
Comment (xiesh1982) [reply cnho_stables/assetlist.json:40]: ok,i remove it later

File: cnho_stables/assetlist.json (hunk 2 above)
Comment (xiesh1982) [reply cnho_stables/assetlist.json:31]: ok,i follow the rule and change it to cnho-stables

Comment (xiesh1982) [reply cnho_stables/assetlist.json:31]: ok,i follow the rule and change it to cnhostables

File: cnho_stables/assetlist.json (hunk 3 above)
Comment (xiesh1982) [reply cnho_stables/assetlist.json:3]: ok ,copy

**Pull request closed without merging**